lib/NativeCode/DynamicLibraryLoaderHelper/DynamicLibraryLoaderHelper.sln

```diff
@@ -39,6 +41,12 @@ Global
 		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Debug|x86.Build.0 = Debug|Win32
 		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Release|x64.ActiveCfg = Release|x64
 		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Release|x86.ActiveCfg = Release|Win32
+		{B3ADEB07-B668-482E-A5AA-9671CE65C5F7}.Debug|x64.ActiveCfg = Debug|x64
+		{B3ADEB07-B668-482E-A5AA-9671CE65C5F7}.Debug|x64.Build.0 = Debug|x64
+		{B3ADEB07-B668-482E-A5AA-9671CE65C5F7}.Debug|x86.ActiveCfg = Debug|x86

```

Comment:
I don't understand these configuration things, so you may already have considered this; it looks like "x86" builds use "Win32" above, but use "x86". Is that intentional?

Reply:
It is indeed intentional.

Reply:
The reasons are weird and complicated - but there are portions of the project that are Win32 - but need to be in a directory marked 'x86'.